ui/app/src/modules/Content/Delete/DeleteDialog.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2007-2020 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { Item } from '../../../models/Item';
+import DeleteDialogUI from './DeleteDialogUI';
+
+const dialogInitialState: any = {
+  submissionComment: '',
+  selectedItems: null
+};
+
+interface DeleteDialogProps {
+  items: Item[];
+
+  onClose?(response?: any): any;
+
+  onSuccess?(response?: any): any;
+}
+
+function DeleteDialog(props: DeleteDialogProps) {
+  const {
+    items,
+    onClose,
+    onSuccess
+  } = props;
+
+  return (
+    <DeleteDialogUI
+      items={items}
+      open={true}
+      onClose={onClose}
+    />
+  )
+}
```

Comment:
Keep an eye on how this component evolves. If it just literally wraps the UI without any container logic, then it's not worth keeping. I do anticipate this one having the container logic (e.g. pulling stuff from redux).

DeleteDialog & DeleteDialogUI can both be in this file.